packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts

```diff
@@ -1,6 +1,8 @@
 import { IButtonStyles } from './Button.Props';
 import { ITheme } from '../../Styling';
 
+const MS_HIGHCONTRAST_ACTIVE = '@media screen and (-ms-high-contrast: active)';
```

Comment:
this const should be in a shared file somewhere, no?

Reply:
Good timing on this, David just added this to CommonStyles. I've changed over every other instance of this to use the one defined there.